.github/workflows/docker.yml

```diff
@@ -19,8 +19,12 @@ jobs:
         env:
```

Comment:
Lines 18-27 (your Docker login step) might not work, or maybe it might. I know there's a lot of ways to do this.

This is how I log into quay:

```
      - name: Login to Quay
        uses: docker/login-action@v2
        with:
          registry: quay.io
          username: ${{ secrets.QUAY_USERNAME }}
          password: ${{ secrets.QUAY_PASSWORD }}
```